Kconfig.headers

```diff
@@ -0,0 +1,36 @@
+# Configurations for headers
+# This file can't be added like other Kconfig files since putting it under include/ will make
+# it visibile to IDEs when writing 'include' statements.
+
+# Copyright (c) 2022 Google Inc
+# SPDX-License-Identifier: Apache-2.0
+
+config MATH_UTIL_FLOAT
```

Comment:
I'm pretty skeptical of this Kconfig option after having seen the results where the math may work only in one case and not the other.